docs/puma_support.md

```diff
@@ -0,0 +1,66 @@
+# Puma Support Policy
```

Comment:
We have bundler version specific code too, should add a policy for that as well.

```
    if bundler_ver < Gem::Version.new('2.1.0')
      Bundler.with_clean_env { yield }
    else
      Bundler.with_unbundled_env { yield }
    end
```